### PR TITLE
Update to shotover 0.6

### PR DESCRIPTION
**Pull request by rukai**
Update to shotover 0.6 and replace redis with valkey

### CODE REVIEW

File: redis-backup-cluster/docker-compose.yaml
Comment (ronycsdu) [inline redis-backup-cluster/docker-compose.yaml:1]: Are we planning to rename the folders later?

File: tests/tests/redis/mod.rs [hunk 1]
```diff
@@ -12,22 +12,22 @@ pub async fn assert_bytes(cmd: &mut Cmd, connection: &mut MultiplexedConnection,
...
+pub async fn valkey_single_connection(address: &str) -> redis::aio::MultiplexedConnection {
     let client = redis::Client::open(address).unwrap();
     client.get_multiplexed_tokio_connection().await.unwrap()
 }
 
 #[tokio::test(flavor = "multi_thread")]
 #[serial]
-async fn redis_cluster_1_1() {
+async fn valkey_cluster_1_1() {
     let _compose = docker_compose("../redis-cluster-1-1/docker-compose.yaml");
```
Comment (justinweng-instaclustr) [inline tests/tests/redis/mod.rs:28]: Should the folder names be renamed as well?

File: tests/tests/redis/mod.rs [hunk 2]
```diff
@@ -12,22 +12,22 @@ pub async fn assert_bytes(cmd: &mut Cmd, connection: &mut MultiplexedConnection,
...
 
-    let mut connection = redis_cluster_connection("redis://172.16.1.2:6380").await;
+    let mut connection = valkey_cluster_connection("redis://172.16.1.2:6380").await;
```
Comment (justinweng-instaclustr) [inline tests/tests/redis/mod.rs:30]: Should this be `valkey:`?

File: redis-backup-cluster/docker-compose.yaml
Comment (rukai) [reply redis-backup-cluster/docker-compose.yaml:1]: yeah, I thought it might mess up the diff.

File: tests/tests/redis/mod.rs (hunk 2 above)
Comment (rukai) [reply tests/tests/redis/mod.rs:30]: The driver doesnt support valkey prefix yet

**Review by justinweng-instaclustr — APPROVED** [review]

**Review by ronycsdu — APPROVED** [review]

**Pull request merged**